app/controllers/api/v1x0/icons_controller.rb

```diff
@@ -42,7 +42,7 @@ def icon_params
       end
 
       def icon_patch_params
-        params.permit(:content, :source_ref, :source_id, :portfolio_item_id)
+        params.permit(:content, :source_ref, :source_id, :portfolio_item_id, :id)
```

Comment:
This is going to go away right? Once we have openapi validation it will handle what should and shouldn't be read-only vs updateable?